common-app-lib/src/cmr/common_app/cache/cubby_cache.clj

```diff
@@ -21,7 +25,11 @@
 ;; Implements the CmrCache protocol by saving data in the cubby application
 (defrecord CubbyCache
   [;; A context containing a connection to cubby
-   context-with-conn]
+   context-with-conn
+
+   ;; A collection of keys used by this cache. Only these keys will be deleted from the backend
+   ;; store on calls to reset
+   keys-to-track]
```

Comment:
My only question would be if this has any impact on existing usage ... I can only assume either "no" or you've fixed it below :-) I'll keep reading ...

Reply:
Had a quick pvt chat about this; looks good.